es-app/src/guis/GuiGameScraper.h

```diff
@@ -14,7 +14,7 @@ class GuiGameScraper : public GuiComponent
 	void onSizeChanged() override;
 
 	bool input(InputConfig* config, Input input) override;
-	void update(int deltaTime);
```

Comment:
Compiler warnings:

In file included from /home/devel/temp/EmulationStation_Stable/es-app/src/guis/GuiMetaDataEd.cpp:10: 
/home/devel/temp/EmulationStation_Stable/es-app/src/guis/GuiGameScraper.h:17:7: warning: 'update' ove
rrides a member function but is not marked 'override' [-Winconsistent-missing-override]
        void update(int deltaTime);
             ^
/home/devel/temp/EmulationStation_Stable/es-core/src/GuiComponent.h:33:15: note: overridden virtual f
unction is here
        virtual void update(int deltaTime);
                     ^